lib/nghttp2/CMakeLists.txt

```diff
@@ -60,7 +60,8 @@ set(OPENSSL_MSVC_STATIC_RT  ON)
 find_package(Python3 COMPONENTS Interpreter)
 
 # Auto-detection of features that can be toggled
-find_package(OpenSSL 1.0.1)
+## This auto detection causes weird linker errors. So, it's disabled for now.
+# find_package(OpenSSL 1.0.1)
```

Comment:
Does this mean we would have a hard requirement on OpenSSL 1.0 if included? That doesn't seem like a good idea and not true for all targets either now.

Reply:
This means that we don't need to link and use OpenSSL on the nghttp2 context. This is because we already wrapped the TLS context via flb_tls. So, I intended to use nghttp2 context w/o TLS.

Reply:
OK, seems reasonable to me and a better solution anyway but I'll leave it to the codeowners to confirm.